packages/contentstack-export/src/export/modules-js/content-types.js

```diff
@@ -18,9 +19,9 @@ class ContentTypesExport {
       this.qs.uid = { $in: this.exportConfig.contentTypes };
     }
     this.contentTypesPath = path.resolve(
-      exportConfig.data,
-      exportConfig.branchName || '',
-      this.contentTypesConfig.dirName,
+      sanitizePath(exportConfig.data),
+      sanitizePath(exportConfig.branchName) || '',
+      sanitizePath(this.contentTypesConfig.dirName),
```

Comment:
@cs-raj , I don't think we need to sanitize branchName and dirName.

Reply:
Yes, but they are being passed to resolve that why added. Even though they are set from the config